### PR TITLE
Amaroc 841 localized base height calculation

### PR DESCRIPTION
**Pull request by regDaniel**
**Description:**

Please add a brief summary of what this PR does.

**Error(s) fixed:**

Please tag them using `#`, e.g. `Fixed #42`.

**Checklists**:
- [x ] New code includes dedicated tests.
- [x] New code has been linted.
- [x] New code follows the project's style.
- [x] New code is compatible with the 3-Clause BSD license.
- [x ] CHANGELOG has been updated.
- [ ] AUTHORS has been updated.
- [ ] Copyright years in module docstrings have been updated.

### CODE REVIEW

File: test/ampycloud/test_data.py [hunk 1]
```diff
@@ -413,3 +413,60 @@ def test_layering_dualeval():
...
+        mock_layer, prms={'CEILOS_FOR_BASE_HEIGHT_CALC': 'ceilo1'}
+    )
+    chunk_filtered.find_slices()
+    chunk_filtered.find_groups()
+    chunk_filtered.find_layers()
+
+    # Since the SD of the two layers is 2ft, it's highly unlikely that the following condition
+    # 1) is met if the filtering works (310+/-2 ft)
+    # 2) is not met if we don't go to the fallback behaviour 460 +/-2 ft)
+    assert chunk_filtered.layers.loc[0, 'height_base'] > 440.
```
[the remark this answers is not in the document]
Comment (regDaniel) [reply test/ampycloud/test_data.py]: Done.

File: src/ampycloud/prms/ampycloud_default_prms.yml [hunk 2]
```diff
@@ -38,6 +38,11 @@ BASE_LVL_HEIGHT_PERC: 5
 # Set to 100 to use all the points.
 BASE_LVL_LOOKBACK_PERC: 100
 
+# If ceilo Ids are set, for any given layer/ group/ slice, only ceilos with the corresponding ID
+# will be used for base height calculation. If the selection results in an empty layer/ group/
+# slice, we go back to using all ceilo hits and ignore the selection.
```
Comment (fpavogt) [inline src/ampycloud/prms/ampycloud_default_prms.yml:43]: I would add here an typical use-case for this parameter. Users may wonder why that parameters could be useful.

**Review by fpavogt — APPROVED** [review]
Looks functional to me. I would add a bit more info to the docs, to help users understand why/when this parameter is useful, and what is the intended usage.

Side-question: should one add a page to the online docs to list all the major changes since v2.0.0 ? I.e. anything major that is not described in the ampycloud article ?

**Review by MericRed — APPROVED** [review]
Fine for me, thanks.

File: setup.py
[the remark this answers is not in the document]
Comment (regDaniel) [reply setup.py:1]: Good idea! I opened #139 and there is a corresponding Jira ticket on AMAROC side.

File: src/ampycloud/prms/ampycloud_default_prms.yml (hunk 2 above)
Comment (regDaniel) [reply src/ampycloud/prms/ampycloud_default_prms.yml]: Done, along with adding a documentation for changes since v2.0.0

**Pull request merged**